tests/sequential/opentelemetry_metrics_cloudevent_source/opentelemetry_metrics_cloudevent_source_test.go

```diff
@@ -0,0 +1,334 @@
+//go:build e2e
```

Comment:
Please, include these test cases inside the already existing cases. Sequential tests are executed one by one, so we should try to add new case files when it's truly needed and currently we are extending prometheus/otel cases